blocks/map/map.css

```
@@ -0,0 +1,192 @@
+    background-repeat: var(--bg-img-repeat);
+    background-position: var(--bg-img-position);
+    background-attachment: var(--bg-img-attachment);
+    background-size: var(--bg-img-size);
+}
+
+.grid-container {
+    inline-size: 1440px;
+    max-inline-size: 96vw;
+    padding-inline: 0 0;
+    margin-inline: auto auto;
+    color: #fff;
+    background-color: transparent;
+    text-align: center;
+}
+
+
+.map-content-container:not(.section) {
+    inline-size: 1440px;
+    max-inline-size: 96vw;
+    margin-inline: auto auto;
+    color: #fff;
+    background-color: transparent;
+    padding-inline: 0.5rem 0.5rem;
+    text-align: center;
+    padding: 0
+}
+
+.map.block:not(.section) {
+    background-color: transparent;
+    padding: 0
+}
+
+hr.separator {
+    border-block-start: none;
+    inline-size: 40px;
+    block-size: 2px;
+    background-color: #0079c1;
+}
+
+hr.center, .text-center hr {
+    margin-inline: auto;
+}
+
+#map-default-content-wrapper > div > h2    { 
+    font-weight: var(--calcite-font-weight-normal);
+    margin: 0 0 var(--space-4);
+    line-height: 1.375;
+}
+
+.btn-white {
+    background: var(--calcite-ui-foreground-1);
+    border: 1px solid var(--calcite-ui-foreground-1);
+    color: var(--calcite-ui-text-2);
+    margin-block: 0 24px;
+}
+
+.btn {
+    --calcite-button-padding-y: 7px;
+    --calcite-button-padding-x: 11px;
+
+    position: relative;
+    display: inline-block;
+    padding: var(--calcite-button-padding-y) var(--calcite-button-padding-x);
+    inline-size: auto;
+    border: 1px solid transparent;
+    border-radius: 0;
+    box-sizing: border-box;
+    transition: all .05s linear;
+    cursor: pointer;
+    white-space: initial;
+    font-size: var(--calcite-font-size--1);
+    line-height: 1.55rem;
+}
+
+.btn-white:is(:hover,:focus) {
+    background: var(--calcite-ui-foreground-2);
+    border-color: var(--calcite-ui-foreground-2);
+    color: var(--calcite-ui-text-2);
+}
+
+.map-frame-aspect-ratio {
+    aspect-ratio: auto 9 / 16;
+}
+
+#map-frame {
+    inline-size: 100%;
+    block-size: 100vh;
+    border: none;
+}
+
+#map-text-content {
+    margin-block: 0 var(--space-4);
+}
+
+#frame-wrapper.is-fullscreen {
+    inset-block-start: 0%;
+    inset-inline-start: 0%;
+    margin: 0;
+
+    /* block-size: 100%; */
+    inline-size: 100%;
+    max-inline-size: 100vw;
+    block-size: 100%;
+    position:fixed;
+    padding: 0;
+    z-index: 11110;
+}
+
+.return-btn {
+    position: fixed;
+    inset-block-start: 16px;
+    inset-inline-start: 50%;
+    display: none;
+    z-index: 11111;
+    transform: translate(-50%);
+    border-radius: 0%;
+}
+
+.return-btn.btn-vis{
+    display: block;
+}
+
+@media (width <= 480px) {
+    #iframe-container {
+        inline-size: 1440px;
+        max-inline-size: 96vw;
+    }
+
+    .map-content-container {
+        margin-inline: 0 0;
+        max-inline-size: 96vw;
+    }
+}
+
+@media (width > 481px) and (width < 1024px) {
```

You should probably only specify the lower margin, since this keeps the code cleaner.

```suggestion
@media (width > 480px) {
```